topics/dev/tutorials/tool-annotation/tutorial.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: tutorial_hands_on
+title: Annotation of a tool with EDAM ontology terms using bio.tools
+level: Introductory
+subtopic: tooldev
+questions:
+- How are Galaxy tools linked to EDAM ontology?
+- How to connect Galaxy tools to bio.tools?
+objectives:
+- Identify Galaxy tools without bio.tools entry
+- Create a bio.tools entry
+- Update a bio.tools entry 
+- Add EDAM ontology terms to a bio.tools entry
+- Link a Galaxy tool to its corresponding bio.tools entry
+time_estimation: 1H
+key_points:
+- Galaxy tools can get EDAM ontology terms from bio.tools
+- bio.tools entry can be created and modified to provide the best EDAM annotations
```

Comment:
Maybe we can add a point here?

- An up-to-date bio.tools entry provides readily accessible metadata that can help users to find and understand tools that are available on Galaxy

Reply:
Please consider using the ["Suggestion Mode" feature of GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) (see step 6). 

By providing a suggestion using the proper suggestion mode:

1. For authors, it is **unambiguous** what you are proposing 
2. It's *also* easier for them to simply accept the suggestion, PR authors prefer suggestions!
3. You get credited in the Git commit helping us properly track attribution